build/config/PublishData.json

```diff
@@ -41,6 +41,13 @@
             "nuget": [ "https://dotnet.myget.org/F/roslyn-for-vs-for-mac/api/v2/package" ],
             "vsix": [ "https://dotnet.myget.org/F/roslyn-for-vs-for-mac/vsix/upload" ],
             "channels": [ "vs-for-mac" ]
+        },
+        "features/AsyncCompletion": {
+            "nugetKind": "PerBuildPreRelease",
+            "version": "2.9.*",
+            "nuget": [ "https://dotnet.myget.org/F/roslyn-for-vs-for-mac/api/v2/package" ],
```

Comment:
That feed might still be consumed by other teams. Please make a new feed.

Reply:
I verified earlier with @dpoeschl that he added a unique pre-release moniker to this branch as well. It shouldn't conflict with other packages.

Reply:
no no, let's use that feed. Nobody's using it.

Reply:
@jasonmalinowski @KirillOsenkov - I don't have a strong opinion about which feed to use, I just want Kirill to be able to do his job and keep streams from crossing where needed. Jason seemed to think this feed is used by TypeScript (and someone else?), so Kirill are you okay with using a separate feed, or does it actually make it easier to use the existing one? Thanks!

Reply:
Let's use this feed.

1. We already have too many feeds in our nuget.config, and we'd have to add a new feed in 3 different places, go through PRs, CI, etc. 
2. This feed is not used by anyone currently. It was used by VS Mac at some point (to consume that version of Roslyn that's published there)
3. This feed is ideal for this particular case. It's Roslyn for VS Mac.

Since the packages will have date-based version numbers on them, I'm curious why the concerns Jason. What kind of breakage would you expect even in theory if we just used this feed?

Reply:
We've verified that F# and TypeScript, both of which also used this feed, aren't using it anymore? In that case, I'm OK with it.

(I'm seeing many wrong statements on this thread which assumed only VS for Mac's repo was pointed to it. That was most definitely not the case. :smile:)

Reply:
And yes, people all the time see new packages on a feed, and think they should upgrade to them. If that gives them magic surprises of new experimental things, it isn't good. You laugh, but you aren't the one who gets to deal with the fallout!

Reply:
Yes I get to deal with the fallout. Our feeds have 3.0 packages presently. I don't see how this makes the problem any worse.

Roslyn is the only repository that insists on creating multiple feeds for items like this.

Reply:
Why did F# and TypeScript use our feed then? The name says VS Mac in it! I would also hope that anytime anyone updates to a newer set of Roslyn packages, they're responsible for verifying that is a good set for them and they should be prepared to deal with the consequences. Also the situation you describe seems hypothetical, while our business need is real and urgent.

Regardless of all that, we need these packages soon, please. Let's try to avoid debating further and just solve the problem of delivering us the packages.

If you have a strong preference which feed to publish them to, go ahead, I respect that. Just account for the fact that if you publish them to a different feed you will cause non-trivial work for our team, I just wanted to make sure the reasons for that extra work would be something more concrete than FUD.

Reply:
> Why did F# and TypeScript use our feed then? The name says VS Mac in it!

The work for you required us to move APIs around which created breaking changes for them. In order to do the coordinated merge of everything, they needed to prep insertions against the new surface area so we could do a simultaneous insertion of everybody's packages.

It's quite possible they've moved off. I chatted with @dpoeschl over lunch to verify we're good (or at least those teams know _not_ to pull new packages), and then I'm OK with unblocking. We can totally use this feed, it's just not something we can do without other people knowing too.